change-the-middle-management-mindset.md

```diff
@@ -0,0 +1,83 @@
+* OKR/cascading KPIs (proven solution, known to work)
+    - OKR are bigger picture than KPIs (defined measurement)
+    - need accountability that transcends from top level goals
+    - existing goals don't have a cross-BU aspect the way IS does. If we get that, then I can honestly speak to what I'm doing as an individual developer
+    - shows actual proof of top down support
+* Events like three day hackathons where the top down people tell middle-managers that their developers have three free days (middle-managers can't say they don't have support)
+    - ShipIt Day has a competitive nature (people choose the one idea that is best; everyone else walks away with nothing). Shows an appetite, but not extended out to everybody perpetually. A bit self-defeating.
+    - We have innovation days (10% time set aside on calendars, broadcast, you can participate). Choose amongst a few and you get three days to form a new team, work on things together. Hackathon for customer-facing or internal projects (prototypes). Could have an IS day where the metric is: Did you work on someone else's code base? Stand it up and then it becomes available for others to use. Come over to collaborate. Opening an issue=1 point; filing a PR= 5 points; 20 points for merging a PR. Then have an award (two weeks off at Hawaii, all paid with your family; or 3-6 months off to work on your own innovative products; or 6 months part time). Every year could have 1-10 people for IS awards--recognition from the CEO.
+* Fellows program if you can achieve guest TC; you get one day a week to do cross-platform work (they report on it as a measurement of success rather than a measurement of loss). Planning is good as a part of resource allocation (might have to change expectations; we're one team, a whole group).
+* Champions program recruited from any group: could be a Middle-Management champion who has gone through the process before. See who are the new believers. Could put badges on people's names in the directory. Could do cardboard cut-outs of the champion with an IS t-shirt ;-)
+
+* Built-into the whole design of IS process. Delivery teams don't own any code themselves; they change code amongst the product teams. Middle Management for product teams know that requests will come in for their code to be changed. They need to have their developers provide mentorship to the delivery team developers. We have different BLs; each is represented by a delivery team. Product teams focus on larger architectural decisions, ensure that the delivery teams don't mess too much with the Products.
+    - We had globalization teams responsible for countries; had constraints (compliance) for each country. Those teams are always good about IS. But for some reason they went away (restructuring).
+
+* Architectural solutions
+    - Microservices architecture: creates incentive organizationally for people to collaborate. If a bug occurs, then it creates a problem for the users. 
```

Comment:
Would it make sense to mention Conway's Law here? To me, it would mean that the architecture of the product (microservices) and the the architecture of the organisation building the product must be in sync. This bullet point makes it sound like it would be enough to switch to microservices.